shell/platform/darwin/graphics/ios_context.h

```diff
@@ -12,7 +12,7 @@
 #include "flutter/fml/macros.h"
 #include "flutter/fml/platform/darwin/scoped_nsobject.h"
```

Comment:
The macOS code uses ARC; you can't share any `scoped_nsobject` headers with macOS at this point (making this work with ARC is non-trivial; I can find PR discussion later to point you to). I would strongly prefer we not put any scroped_nsobject code in a place where we intended to share with macOS.